drivers/modem/Kconfig.quectel-bg9x

```diff
@@ -0,0 +1,64 @@
+# Quectel BG9x driver options
+
+# Copyright (c) 2020 Analog Life LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config MODEM_QUECTEL_BG9X
+	bool "Enable quectel modem driver"
+	select MODEM_CONTEXT
+	select MODEM_CMD_HANDLER
+	select MODEM_IFACE_UART
+	select MODEM_SOCKET
+	select NET_OFFLOAD
+	select NET_SOCKETS_OFFLOAD
+	imply GPIO
+	help
+	  Choose this setting to enable quectel BG9x LTE-CatM1/NB-IoT modem
+	  driver.
+
+config MODEM_QUECTEL_BG9X_RX_STACK_SIZE
+	int "Stack size for the quectel BG9X modem driver RX thread"
+	default 1028
+	help
+	  This stack is used by the quectel BG9X RX thread.
+
+config MODEM_QUECTEL_BG9X_RX_WORKQ_STACK_SIZE
+	int "Stack size for the quectel BG9X modem driver work queue"
+	default 2048
+	help
+	  This stack is used by the work queue to pass off net_pkt data
+	  to the rest of the network stack, letting the rx thread continue
+	  processing data.
+
+config MODEM_QUECTEL_BG9X_APN
+	string "APN for establishing network connection"
+	default "hologram"
```

Comment:
This value seems to be specific to your network. Perhaps `internet` would be more generic default value here